Dockerfile

```diff
@@ -18,7 +18,7 @@ ENV RHOAS_CLI_PATH="/usr/local/bin/rhoas"
 ENV OC_CLI_PATH="/usr/local/bin/oc"
 
 # Install required packages
-RUN microdnf install shadow-utils yum
+RUN microdnf install shadow-utils jq
```

Comment:
Needs a rebase - I have added JQ in a previous PR: https://github.com/redhat-developer/app-services-tools/blob/main/Dockerfile#L27